docs/app/pages/components/components-sections/input-controls/typeahead/snippets/app.html

```diff
@@ -0,0 +1,48 @@
+<div class="row">
+    <div class="col-md-12 form-group-validate has-feedback">
+        <form>
+            <input [value]="value"
+                   [(ngModel)]="input"
+                   [ngModelOptions]="{standalone: true}"
+                   [placeholder]="'Enter Text'"
+                   class="form-control"
+                   (keydown.enter)="$event.preventDefault()"
+                   (click)="dropdown(); $event.stopPropagation();"
+                   (keydown)="dropdown(); typeaheadKeyService.handleKey($event, typeahead);
+                   $event.stopPropagation();">
+            <ux-typeahead #typeahead
+                          class="typeahead-example"
+                          [open]="dropdownOpened"
```

Comment:
Consider renaming to `dropdownOpen` which indicates its current open state, not if it has been opened at some point

Reply:
renamed